### DISCUSSION TITLE
no console.log output in runtime logs

### DISCUSSION THREAD

**Discussion by slowlynorge** [vercel/community#7190] [Bug Report]
### Page to Investigate

my runtime logs page

### Steps to Reproduce

i have console.log all over my code, but they do not appear in runtime logs. only console.error and actual errs appear in the log. never any info or log level messages.
<img width="767" alt="image" src="https://github.com/vercel/community/assets/92298903/468e9d49-3ca3-4b26-8ad8-802690a7900b">

**Comment by Ilaay** [comment]
Same problem

[a comment by an automated account is not shown]
